From ad96b1824d013d3c254a344a7fda715b0b991778 Mon Sep 17 00:00:00 2001
From: Sergey Temerkhanov <s.temerkhanov@gmail.com>
Date: Tue, 3 May 2016 18:56:26 -0700
Subject: [PATCH 0394/1239] thunderx: Remove obsolete SATA-related code

This code is unused since moving AHCI to PCI. This commit
removes the obsolete code

Signed-off-by: Sergey Temerkhanov <s.temerkhanov@gmail.com>
---
 board/cavium/thunderx/sata.c | 47 ------------------------------------
 1 file changed, 47 deletions(-)
 delete mode 100644 board/cavium/thunderx/sata.c

diff --git a/board/cavium/thunderx/sata.c b/board/cavium/thunderx/sata.c
deleted file mode 100644
index 478cce08fc..0000000000
--- a/board/cavium/thunderx/sata.c
+++ /dev/null
@@ -1,47 +0,0 @@
-/** @file
-#
-#  Copyright (c) 2014, Cavium Inc. All rights reserved.<BR>
-#  This program and the accompanying materials
-#  are licensed and made available under the terms and conditions of the BSD License
-#  which accompanies this distribution.  The full text of the license may be found at
-#  http://opensource.org/licenses/bsd-license.php
-#
-#  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
-#
-#**/
-
-#include <common.h>
-#include <ahci.h>
-#include <scsi.h>
-#include <sata.h>
-#include <asm/io.h>
-#include <cavium/atf.h>
-#include <cavm-csr.h>
-
-
-#define AHCI_PER_NODE 16
-
-uintptr_t sata_baseaddress(int dev)
-{
-	union satax_uctl_ctl uctl_ctl;
-	int node = dev / AHCI_PER_NODE;
-
-	dev %= AHCI_PER_NODE;
-
-	if (node >= atf_node_count())
-		return 0;
-
-	uctl_ctl.u = readq(CSR_PA(node, SATAX_UCTL_CTL(dev)));
-
-	debug("sata %d at node %d: uctl_ctl.u: %llx\n",
-	      dev, node, uctl_ctl.u);
-
-	if (!uctl_ctl.s.sata_uahc_rst &&
-		!uctl_ctl.s.sata_uctl_rst &&
-		uctl_ctl.s.a_clk_en) {
-		return CSR_PA(node, SATAX_PF_BAR0(dev));
-	} else {
-		return 0;
-	}
-}
-- 
2.29.0

